src/Notes/WC_Admin_Note.php

```diff
@@ -23,7 +23,7 @@ class WC_Admin_Note extends \WC_Data {
 	const E_WC_ADMIN_NOTE_INFORMATIONAL = 'info';    // used for presenting informational messages.
 
 	// Note status codes.
-	const E_WC_ADMIN_NOTE_PENDING    = 'pending';    // the note is not actioned but shouldn't be displayed.
+	const E_WC_ADMIN_NOTE_PENDING    = 'pending';    // the note is pending - hidden but not actioned.
 	const E_WC_ADMIN_NOTE_UNACTIONED = 'unactioned'; // the note has not yet been actioned by a user.
 	const E_WC_ADMIN_NOTE_ACTIONED   = 'actioned';   // the note has had its action completed by a user.
 	const E_WC_ADMIN_NOTE_SNOOZED    = 'snoozed';    // the note has been snoozed by a user.
```

Comment:
Looks like we'll need to add "preunactioned" here - I'm getting this error in testing:

```
PHP Fatal error:  Uncaught WC_Data_Exception: The admin note status prop (preunactioned) is not one of the supported statuses. in /srv/www/wordpress-default/public_html/wp-content/plugins/woocommerce/includes/abstracts/abstract-wc-data.php:810
Stack trace:
#0 /srv/www/wordpress-default/public_html/wp-content/plugins/woocommerce-admin/src/Notes/WC_Admin_Note.php(450): WC_Data->error('admin_note_inva...', 'The admin note ...')
```

Reply:
Also:
```
PHP Fatal error:  Uncaught WC_Data_Exception: The admin note icon prop cannot be empty. in /srv/www/wordpress-default/public_html/wp-content/plugins/woocommerce/includes/abstracts/abstract-wc-data.php:810
Stack trace:
#0 /srv/www/wordpress-default/public_html/wp-content/plugins/woocommerce-admin/src/Notes/WC_Admin_Note.php(412): WC_Data->error('admin_note_inva...', 'The admin note ...')
```

Reply:
Whoops - I had old data. Disregard all of this! 😅